go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/companieshouse/go-sdk-manager v0.1.12
 	github.com/companieshouse/go-session-handler v0.1.5
 	github.com/companieshouse/gofigure v0.1.4
-	github.com/companieshouse/lfp-pay-api-core v1.0.7
+	github.com/companieshouse/penalty-payment-api-core v1.0.7
```

Comment:
Does a rename precipitate a new version in pipeline for this? Would it be bumped to 1.0.8?

Reply:
yes I will update this